dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/EntryVersionHelper.java

```diff
@@ -462,6 +464,26 @@ default SortedSet<SourceFile> getVersionSourceFiles(Long entryId, Long versionId
         return new TreeSet<>(sourceFiles);
 
     }
+
+    static Optional<Version> determineRepresentativeVersion(Entry entry) {
+        return Optional.ofNullable(entry.getActualDefaultVersion())
+            .or(() -> determineRepresentativeVersion(entry.getWorkflowVersions()));
+    }
+
+    static <V extends Version> Optional<V> determineRepresentativeVersion(Set<V> versions) {
```

Comment:
This file looks a bit notorious for lacking documentation, but would still be nice to add it for these new methods at least.